src/pages/ContactPage.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useEffect } from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import BaseLayout from '../layout/BaseLayout';
+
+import '../styles/ContactPage.scss';
+
+type Props = {
+  location: {
+    state: {
+      accession?: string;
+    }
+  }
+} & RouteComponentProps;
+
+const ContactPage: React.FC<Props> = ({
+  location: { state: { accession = '' } },
+}) => {
+  const [subject, setSubject] = useState('');
+  const [agree, setAgree] = useState(false);
+
+  useEffect(() => {
+    if (accession.length > 0) {
+      setSubject(`Covid-19 UniProtKB/Swiss-Prot ${accession} entry update request`);
+    }
+  }, [accession]);
+
+  return (
+    <BaseLayout>
+      <form action="https://www.uniprot.org/feedback" method="POST">
+        <input type="hidden" name="entry" value={accession} />
+        <div className="contact-page-form">
+          <h3>Contact</h3>
+          <div className="contact-page-form__row">
+            <span>
```

Comment:
It looks weird on a wide screen - I think you can use the foundation form styling oh having "Name" in a `<label/>` and it will position it nicely above the input.

![image](https://user-images.githubusercontent.com/3329765/78144335-624d4880-7427-11ea-9562-dc52f675f2ab.png)

Reply:
I initially used Labels but it wouldn't look like the contact form from uniprot.org or other forms in the new uniprot website. That's why I started creating an Input element in Franklin, because it looks like all forms in the new uniprot website also are done slightly different from each other and with different CSS too.

This is now using labels.